src/components/2019/year-in-review/News.js

```diff
@@ -0,0 +1,534 @@
+                            </StyledStoryLink>
+                            <Date> Mar. 16 </Date>
+                        </div>
+                        <div class="col-md-3 narrow-col-left">
+                            <StyledStoryLink href="https://www.thedp.com/article/2019/07/jerome-allen-probation-accepting-bribes-penn-mens-basketball-college-admissions-scandal-esformes"  >
+                                <SecondaryStory> Former Penn coach Jerome Allen avoids jail time in admissions bribery scandal </SecondaryStory>
+                            </StyledStoryLink>
+                            <Date> July 2 </Date>
+                        </div>
+                    </div>
+                </MoreStories>
+            </div>
+        </Section>
+
+        <Section inverse={true}>
+            <div class="container">
+                <SubSectionHeader>SCHOOL RENAMINGS</SubSectionHeader>
+                <div class="row">
+                    <div class="col-md-6 right-border-inverse narrow-col-right">
+                        <img class="img-fluid"
+                            src="https://s3.amazonaws.com/snwceomedia/dpn/e96e9108-c45b-468e-94c6-dde38d382089.sized-1000x1000.jpg" />
+                        <MainHeader color={"#131313"}>Penn Design and Penn Law renamed after large donations</MainHeader>
+                        <SectionMainText inverse={true}>Penn renamed two of their twelve schools this year after receiving large donations. In February, Penn’s School of Design was renamed the University of Pennsylvania Stuart Weitzman School of Design after the fashion icon and 1963 Wharton alumnus. Students voiced anger over the renaming, arguing that Weitzman’s career as a designer has no relation to any academic program at Penn. Students added administrators should have considered student input before the decision. In November, the University renamed Penn Law School to the University of Pennsylvania Carey Law School after the largest donation to any law school. A petition demanding the school change its short-form name from “Carey Law” back to “Penn Law” garnered over 500 signatures from students and alumni within two days of the announcement. Ten days after the initial announcement of the name change, Penn Law Dean Ted Ruger announced the school will change its short-form name back to “Penn Law” after widespread backlash from students and alumni.</SectionMainText>
+                    </div>
+                    <div class="col-md-6 narrow-col-left">
+                        <div class="row">
+                            <div class="col-md-6">
+                                <a href="https://www.thedp.com/article/2019/10/stuart-weitzman-school-of-design-penn-wharton"><PrimaryStory inverse={true}> ‘We were completely blindsided': PennDesign students frustrated by the school’s renaming </PrimaryStory></a>
+                                    <Date inverse={true}> Oct. 2 </Date>
```

Comment:
same as before with just being able to write "inverse". I'm not going to flag all of these, please go through and refactor yourself :)